src/node_modules/round-contributors/round-contributors-controller.coffee

```diff
@@ -46,8 +56,18 @@ module.exports = ($scope, $state, config, AuthService, AllocationModel, Allocati
     debug("upload success", response)
     # TODO display success in alerts (see group-members)
     $state.reload('layout')
+    # i want to add an alert after everything is reloaded, have no idea how
+    # $scope.addAlert(
+    #   type: "ya brah",
+    #   msg: "everyone added brah"
+    # )
```

Comment:
i wonder if there's an issue where the reload causes a new controller to come into existence with a new scope, so modifying the old scope isn't doing anything as that controller is no longer part of the current view.

i can think of a hack to test this, where in the .then of the reload you broadcast an event over the root scope, which you listen to somewhere else in the controller, so that way the old controller can send a message to the new controller in order to add the alert.

[how to broadcast, afaik](http://stackoverflow.com/a/19446975)